dataprofiler/data_readers/data_utils.py

```diff
@@ -232,7 +260,7 @@ def read_json(data_generator, selected_columns=None, read_in_string=False):
                 ),
                 ignore_dicts=True,
             )
-            lines.append(obj)
+            lines.append(cast(Dict, obj))  # should always be Dict
```

Comment:
Is this comment needed? isnt that what the type cast is doing?

Reply:
that's fair ... not really needed per se. I'm fine leaving just for clarity in the code

Reply:
I added it because `cast` is inherently unsafe but it turns out `obj` will always be a `Dict` here.

Reply:
Is this bc of the `OrderedDict`?

Reply:
What's our guarantee that it is a dict?

Reply:
I've determined this could be a list. however, not a string.

Reply:
The code that could result in this would be:
```python
data = dp.Data(StringIO("[1]\n[2]"))
data.head()
```
Currently, there's an error,  in the reader not supporting this. However, it should.

Reply:
should be resolved in PR #691

Reply:
No longer a need for casting from my latest commit.